packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -521,105 +587,78 @@ class _ContrastReport {
   /// the rectangle is empty.
   final Color darkColor;
 
-  /// Whether the rectangle contains only one color.
-  final bool isSingleColor;
-
-  /// Whether the rectangle contains 0 pixels.
-  final bool isEmptyRect;
-
   /// Computes the contrast ratio as defined by the WCAG.
   ///
-  /// source: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
-  double contrastRatio() {
-    return (_luminance(lightColor) + 0.05) / (_luminance(darkColor) + 0.05);
-  }
-
-  /// Relative luminance calculation.
-  ///
-  /// Based on https://www.w3.org/TR/2008/REC-WCAG20-20081211/#relativeluminancedef
-  static double _luminance(Color color) {
-    double r = color.red / 255.0;
-    double g = color.green / 255.0;
-    double b = color.blue / 255.0;
-    if (r <= 0.03928)
-      r /= 12.92;
-    else
-      r = math.pow((r + 0.055)/ 1.055, 2.4).toDouble();
-    if (g <= 0.03928)
-      g /= 12.92;
-    else
-      g = math.pow((g + 0.055)/ 1.055, 2.4).toDouble();
-    if (b <= 0.03928)
-      b /= 12.92;
-    else
-      b = math.pow((b + 0.055)/ 1.055, 2.4).toDouble();
-    return 0.2126 * r + 0.7152 * g + 0.0722 * b;
-  }
+  /// Source: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
+  double contrastRatio() =>
+      (lightColor.computeLuminance() + 0.05) /
+      (darkColor.computeLuminance() + 0.05);
```

Comment:
nit: move `lightColor...` to previous line and re-align

Reply:
Done